localstate/localstate.go

```diff
@@ -62,6 +76,30 @@ func (ls *LocalState) SaveRef(builderName, nodeName, id string, st State) error
 	return ioutils.AtomicWriteFile(filepath.Join(refDir, id), dt, 0600)
 }
 
+func (ls *LocalState) ReadGroup(id string) (*StateGroup, error) {
```

Comment:
Where is this used? What needs to access refs per group ID?

Reply:
This is an helper like `ReadRef` to be consistent.